modules/claims_api/app/controllers/claims_api/v1/forms/intent_to_file_controller.rb

```diff
@@ -27,6 +27,7 @@ def submit_form_0966
           check_for_invalid_burial_submission! if form_type == 'burial'
 
           bgs_response = bgs_service.intent_to_file.insert_intent_to_file(intent_to_file_options)
+          ClaimsApi::Logger.log('itf', status: 'Submitted')
```

Comment:
Your `case` for `itf` no longer works with a `status` attribute.
Instead, it's now a `detail` attribute.
The detail value should be something like, `submitted to BGS`